enroll_batch_students.py

```diff
@@ -0,0 +1,71 @@
+"""
+Sample of file which should be downloaded from Edx course's Instructor Data Download page with students anonymous ids
+
+is located in edx-adapt/data dir.
+"""
+import csv
+from itertools import islice
+import json
+import os
+import sys
+
+import requests
+
+from config import EDXADAPT, DEFAULT_PROBABILITIES, DEFAULT_SKILLS
+
+
+def get_students_for_enrollment(path_to_file):
+    """
+    Parse csv file with student anonymous data and prepare a list of students for enrollment in edx-adapt.
+    """
+    if not path_to_file:
+        print "Usage is: python enroll_batch_student.py /path/to/file/with/student_ids"
+        sys.exit()
+    elif not os.path.exists(path_to_file):
+        print "File does not exist, please try again"
+        sys.exit()
+    students = []
+    with open(path_to_file) as file_with_ids:
+        raw_students_ids = csv.reader(file_with_ids)
+        for line in islice(raw_students_ids, 2, None):
+            students.append(line[1])
+    return students
+
+
+def get_enrolled_students(headers, **kwargs):
+    """
+    Get student already enrolled in edx-adapt.
+    """
+    users = requests.get('http://{HOST}:{PORT}/api/v1/course/{COURSE_ID}/user'.format(**kwargs), headers=headers).json()
+    if users:
+        return set(users['users']['finished']) | set(users['users']['in_progress'])
+    else:
+        print "Course or edx-adapt server is not found"
+        sys.exit()
+
+
+def main(**kwargs):
+    path_to_file = sys.argv[1]
+    students = get_students_for_enrollment(path_to_file)
+    headers = {'Content-type': 'application/json'}
+    enrolled_students = get_enrolled_students(headers, **kwargs)
+
+    for student_id in set(students).difference(enrolled_students):
+        payload = {'user_id': student_id}
+        requests.post(
+            'http://{HOST}:{PORT}/api/v1/course/{COURSE_ID}/user'.format(**kwargs), json=payload, headers=headers
+        )
+        print "student {} is enrolled in course {}".format(student_id, kwargs['COURSE_ID'])
```

Comment:
Since we're using `"".format()` anyway, it also makes sense to use `print()` function too. Here and everywhere.